src/main/java/org/opendevstack/provision/services/JiraAdapter.java

```diff
@@ -60,11 +51,14 @@
   @Value("${jira.permission.filepattern}")
   private String jiraPermissionFilePattern;
 
+  public static final String jiratemplateKeyPrefix = "jira.project.template.key.";
+  public static final String jiratemplateTypePrefix = "jira.project.template.type.";
```

Comment:
I'd keep the casing consistent, so `jiraTemplateKeyPrefix` here and also above `confluenceTemplateKeyPrefix` ...